templates/portal/home.html.twig

```diff
@@ -54,12 +60,52 @@
             });
         }
 
+        const activityMonitor = function () {
+            let timeOut, warnTimeOut, time;
+            if (Number(inactivityTimeOut) < 300) {
+                inactivityTimeOut = 900; // 15 minutes. < 5 minutes is too short.
+            }
+            time = inactivityTimeOut * 1000;
+
+            window.onload = resetLogOutTimer;
+            window.onmousemove = resetLogOutTimer;
+            window.onmousedown = resetLogOutTimer;
+            window.ontouchstart = resetLogOutTimer;
+            window.ontouchmove = resetLogOutTimer;
+            window.onclick = resetLogOutTimer;
+            window.onkeydown = resetLogOutTimer;
+            window.addEventListener('scroll', resetLogOutTimer, true);
+
+            function logoutWarn() {
+                const warnMsg = xl('Your current log on session will expire in 30 seconds.') +
+                    '<br />' + xl('Please take action to prevent being logged out!');
```

Comment:
```
const warnMsg = {{ 'Your current log on session will expire in 30 seconds.' | xlj }} +
                    '<br />' + {{ 'Please take action to prevent being logged out!' | xlj }};
```
then can remove the xl() stub function above